pkg/commands/lmpop.ts

```diff
@@ -0,0 +1,17 @@
+import { Command, CommandOptions } from "./command";
+
+/**
+ * @see https://redis.io/commands/lmpop
+ */
+export class LmPopCommand<TValues> extends Command<
+  [string, TValues[]] | null,
+  [string, TValues[]] | null
+> {
+  constructor(
+    cmd: [numkeys: number, keys: string[], "LEFT" | "RIGHT", count?: number],
+    opts?: CommandOptions<[string, TValues[]] | null, [string, TValues[]] | null>,
+  ) {
+    const [numkeys, keys, direction, count] = cmd;
+    super(["LMPOP", numkeys, ...keys, direction, "COUNT", count], opts);
```

Comment:
`count` is optional, I think we should only add `"COUNT"` when `count` is provided

Reply:
Yes, sorry. Good catch.